tool/site/client/src/pages/sendPreSignPage.tsx

```diff
@@ -1,29 +1,59 @@
-import PrescriptionSummaryView, {createSummaryPrescription} from "../components/prescription-summary/prescriptionSummaryView"
+import PrescriptionSummaryView, {createSummaryPrescriptionViewProps, PrescriptionSummaryErrors} from "../components/prescription-summary/prescriptionSummaryView"
 import * as React from "react"
 import {useCallback, useContext, useEffect, useState} from "react"
 import {useCookies} from "react-cookie"
 import {Bundle, OperationOutcome} from "fhir/r4"
 import LongRunningTask from "../components/longRunningTask"
 import {AppContext} from "../index"
-import {ActionLink, Button, Label} from "nhsuk-react-components"
+import {ActionLink, Button, Form, Label} from "nhsuk-react-components"
 import ButtonList from "../components/buttonList"
 import {isBundle} from "../fhir/typeGuards"
 import {redirect} from "../browser/navigation"
 import {getResponseDataIfValid} from "../requests/getValidResponse"
 import {axiosInstance} from "../requests/axiosInstance"
 import BackButton from "../components/backButton"
+import {Formik, FormikErrors} from "formik"
+import {getMedicationRequestResources} from "../fhir/bundleResourceFinder"
+import {updateBundleIds} from "../fhir/helpers"
 
 interface SendPreSignPageProps {
   prescriptionId: string
 }
 
+interface SendPreSignPageFormValues {
+  numberOfCopies: string
+  nominatedOds: string
+}
+
+type SendPreSignPageFormErrors = PrescriptionSummaryErrors
+
 const SendPreSignPage: React.FC<SendPreSignPageProps> = ({
   prescriptionId
 }) => {
   const {baseUrl} = useContext(AppContext)
-  const [sendConfirmed, setSendConfirmed] = useState<boolean>(false)
+  const [editMode, setEditMode] = useState(false)
+  const [sendPageFormValues, setSendPageFormValues] = useState<SendPreSignPageFormValues>()
   const retrievePrescriptionTask = () => retrievePrescription(baseUrl, prescriptionId)
 
+  const validate = (values: SendPreSignPageFormValues) => {
+    const errors: FormikErrors<SendPreSignPageFormErrors> = {}
+
+    const copiesError = "Please provide a number of copies between 1 and 25"
+    if (!values.numberOfCopies) {
+      errors.numberOfCopies = copiesError
+    } else {
+      const copies = parseInt(values.numberOfCopies)
+      if (isNaN(copies)) {
+        errors.numberOfCopies = copiesError
+      }
+      if (copies > 25) {
+        errors.numberOfCopies = copiesError
+      }
```

Comment:
its the same error in either case so we could just do 
```
      if (isNaN(copies) || copies > 25) {
        errors.numberOfCopies = copiesError
      }
```
I think, this also doesn't cover negative numbers or 0but assuming our users aren't trying to break it I think thats probs fine

Reply:
pushed this change and handled negative number case too